library/library.mld

```diff
@@ -0,0 +1,6 @@
+{1 The Coq library}
+
+This chapter describes the Coq library, which is made of two parts:
+
+- a general mechanism to keep a trace of all operations and of the state of the system, with backtrack capabilities;
+- a global environment for the CCI, with functions to export and import compiled modules.
```

Comment:
what is CCI?

Reply:
This was in the old text, I dunno.

Reply:
ok, then it is the french way of spelling CIC. Please fix

Reply:
I may remove the mld files at all I think, until we have a better plan for the API docs.

Reply:
They were just there to illustrate what you can do, but new odoc will open up some more possibilities so we can wait a bit.